src/timepicker/timepicker.js

```diff
@@ -130,7 +130,6 @@ class TimePicker extends React.Component<TimePickerPropsT, TimePickerStateT> {
       });
       this.setState({
         steps: steps,
-        value: {id: closestStep, label: secondsToLabel(closestStep)},
```

Comment:
this would be a breaking change fyi, may need to add some configuration to opt-out of this behavior